BASE/Test/ServerTelemetryChannel.Test/TelemetryChannel.Nuget.Tests/TelemetryChannel.Nuget.Tests.csproj

```diff
@@ -18,11 +18,6 @@
     <DebugSymbols>true</DebugSymbols> 
   </PropertyGroup>
 
-  <PropertyGroup Condition="$(OS) != 'Windows_NT'">
-    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <TargetFrameworks>netstandard1.3</TargetFrameworks>
-  </PropertyGroup>
-  
   <ItemGroup Condition="$(OS) == 'Windows_NT'">
```

Comment:
I agree with this change, but it's causing the build to fail.
For now, I think you could revert it and just add a comment inside the PropertyGroup:
`<!-- This can be removed. Requires update to .yml -->`

You can create a github issue to investigate a fix later.

Reply:
ok, done: https://github.com/microsoft/ApplicationInsights-dotnet/issues/1847